README.md

```diff
@@ -71,23 +71,24 @@ Yup, I do:
 ```
 Framework            Requests/sec  % from best
 ----------------------------------------------
-rack                     19860.11      100.00%
```

Comment:
Please, do not modify README benchmark numbers, only include new frameworks when listing the scripts but not the numbers as your hardware differs from the one used to run them in the first place.

Reply:
Okay I will revert.